### ISSUE TITLE
Support password-less SSH via config

### ISSUE THREAD

**Issue by zhi-shen** [reconquest/orgalorg#6]
Is there a way this tool can support password-less SSH via config setup? We don't have .ssh/id_rsa.

Reference:

https://www.digitalocean.com/community/tutorials/how-to-configure-custom-connection-options-for-your-ssh-client

**Comment by seletskiy** [comment]
@zhi-shen: How do you authenticate on the remote nodes, then?

**Comment by seletskiy** [comment]
@zhi-shen: Yes, I understand. Basically, you expect that orgalorg will parse `~/.ssh/config` and use specified key from that config.

Unfortunately, it's not yet implemented (I think, I'll do it soon). So now the only possibility is to specify key by yourself or use use wrapper shell-function to do it for you.

**Comment by zhi-shen** [comment]
Thanks. I'll keep an eye out for the feature.

-zhi

**Comment by grumpycatt** [comment]
:+1: It's essential, otherwise it can be an headache to use orgalorg.

**Comment by seletskiy** [comment]
@grumpycatt: I'll implement it in meantime.
